internal/pkg/oasparser/config/config.go

```diff
@@ -23,5 +23,5 @@ const (
 	LISTENER_PORT     uint32 = 10000
 	API_DEFAULT_PORT   uint32 = 8080
 	MANAGER_STATPREFIX    string = "ingress_http"
-	CLUSTER_CONNECT_TIMEOUT    = 5 * time.Second
+	CLUSTER_CONNECT_TIMEOUT    = 20 * time.Second
```

Comment:
These should be taken from a configuration file.

Reply:
that menas config.toml ?  so everything should be moved to config.toml ?

Reply:
Need to move the ones the user should configure...

Reply:
done